internal/xds/translator/extauth.go

```diff
@@ -117,6 +117,14 @@ func extAuthConfig(extAuth *ir.ExtAuth) *extauthv3.ExtAuthz {
 		})
 	}
 
+	if extAuth.BodyToExtAuth != nil {
+		config.WithRequestBody = &extauthv3.BufferSettings{
+			MaxRequestBytes:     1024,
+			AllowPartialMessage: false,
+			PackAsBytes:         false,
+		}
```

Comment:
Arbitrary value (but must be greater than 0) for the `MaxRequestBytes` field, given that the `AllowPartialMessage` field is always equal to false.